governance/handler.go

```diff
@@ -127,37 +156,62 @@ func (g *Governance) AddVote(key string, val interface{}) bool {
 
 func (g *Governance) adjustValueType(key string, val interface{}) interface{} {
 	k := GovernanceKeyMap[key]
-	reqType := GovernanceItems[k].t
 	var x interface{}
 
 	// When an int value comes from JS console, it comes as a float64
-	if reqType == uint64T && reflect.TypeOf(val) == float64T {
-		x = uint64(val.(float64))
-		if float64(x.(uint64)) == val.(float64) {
+	if GovernanceItems[k].checkValueType(uint64(0)) {
+		v, ok := val.(float64)
+		if !ok {
+			return val
+		}
+		x = uint64(v)
+		if float64(x.(uint64)) == v {
```

Comment:
```suggestion
		if float64(x) == v {
```

This would be also okay because x is already the type of uint64.

Reply:
`x` is declared as interface{} type. So, It needs to be converted to `uint64`.

Reply:
okay. I will suggest this in another way.